frontend/src/components/Preload/Preload.js

```diff
@@ -20,17 +22,44 @@ const Preload = ({ instruction, pageTitle, duration, url, onNext }) => {
             onNext();
         }
     };
+
     // Audio preloader
     useEffect(() => {
-        // Load audio until available
-        // Return remove listener
-        return audio.loadUntilAvailable(url, () => {
-            audioIsAvailable.current = true;
-            if (timeHasPassed.current) {
-                onNext();
+        if (playConfig.play_method === 'BUFFER' && !playConfig.external_audio) {
+
+            // Use Web-audio and preload sections in buffers            
+            sections.map((section, index) => {
+                // Clear buffers if this is the first section
+                if (index === 0) {
+                    webAudio.clearBuffers();
+                }
+                
+                // Load sections in buffer
+                // Return remove listener  
+                return webAudio.loadBuffer(section.id, MEDIA_ROOT + section.url, () => {                    
```

Comment:
The comment is not correct, right? What's being returned here is an async request. `useEffect` *should* return its own cleanup, but this `return` is from the `map` method, hence, nothing is returned from the `useEffect` hook itself.